public_html/administrator/components/com_jed/views/reviews/tmpl/default.php

```diff
@@ -17,80 +17,80 @@
 
 /** @var JedViewReviews $this */
 
-$user      = Factory::getUser();
-$userId    = $user->get('id');
-$listOrder = $this->escape($this->state->get('list.ordering'));
-$listDirn  = $this->escape($this->state->get('list.direction'));
-$canOrder  = $user->authorise('core.edit.state', 'com_jed.revews');
-$saveOrder = $listOrder === 't.ordering';
+$user       = Factory::getUser();
+$userId     = $user->get('id');
+$listOrder  = $this->escape($this->state->get('list.ordering'));
+$listDirn   = $this->escape($this->state->get('list.direction'));
+$canOrder   = $user->authorise('core.edit.state', 'com_jed.reviews');
+$saveOrder  = $listOrder === 't.ordering';
 
 HTMLHelper::_('formbehavior.chosen', 'select');
 ?>
 
 <form action="<?php echo Route::_('index.php?option=com_jed&view=reviews'); ?>" method="post" name="adminForm"
       id="adminForm">
 	<?php if (!empty($this->sidebar)) : ?>
-	<div id="j-sidebar-container" class="span2">
+    <div id="j-sidebar-container" class="span2">
 		<?php echo $this->sidebar; ?>
-	</div>
-	<div id="j-main-container" class="span10">
+    </div>
+    <div id="j-main-container" class="span10">
 		<?php else : ?>
-		<div id="j-main-container">
+        <div id="j-main-container">
 			<?php endif; ?>
 			<?php
 			// Search tools bar
 			echo LayoutHelper::render('joomla.searchtools.default', array('view' => $this));
```

Comment:
Use the array short notation `[]`. The comment can go, we can see it is the search tools.